### PR TITLE
Update follow redirects dependency to 1.15.4 for release branch 1.41

### PR DESCRIPTION
**Pull request by SakshiS-harma**
Updating follow-redirects to update it from version 1.15.3 to 1.15.4 to fix a vulnerability.

### CODE REVIEW

File: build/actions/yarn.lock [hunk 1]
```diff
@@ -285,6 +285,8 @@ node-fetch@^2.3.0:
   version "2.6.7"
   resolved "https://registry.yarnpkg.com/node-fetch/-/node-fetch-2.6.7.tgz#24de9fba827e3b4ae44dc8b20256a379160052ad"
   integrity sha512-ZjMPFEfVx5j+y2yF35Kzx5sF7kDzxuDj6ziH4FFbOp87zKDZNx8yExJIb05OGF4Nlt9IHFIMBkRl41VdvcNdbQ==
+  dependencies:
+    whatwg-url "^5.0.0"
```
Comment (corivera) [inline build/actions/yarn.lock:289]: Is this change intentional?

Comment (SakshiS-harma) [reply build/actions/yarn.lock]: Fixed it. Thanks for your due diligence.

**Review by Charles-Gagnon — APPROVED** [review]
